### PR TITLE
Port licensing information to reuse format

### PR DESCRIPTION
**Pull request by CarlSchwan**
See discussion in https://github.com/nextcloud/server/issues/29723

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+<!--
+SPDX-FileCopyrightText: 2021 Frieder Schrempf
+SPDX-FileCopyrightText: 2021 Carl Schwan <carl@carlschwan.eu>
+SPDX-FileCopyrightText: 2021 Cyrille Bollu
+SPDX-FileCopyrightText: 2021 Pierre Ozoux <pierre@ozoux.net>
+SPDX-FileCopyrightText: 2021 Julius Härtl <jus@bitgrid.net>
+SPDX-License-Identifier: AGPL-3.0-or-later
+-->
```
Comment (fschrempf) [inline README.md:8]: Are these tags auto-generated from the git history?
Why are some e-mail addresses missing?
Do we really need copyright and license information in documentation files such as `README.md`? If yes is AGPL the correct license for documentation files or should they be CC0 or something else?

File: .reuse/dep5 [hunk 2]
```diff
@@ -0,0 +1,16 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: profiler
```
Comment (juliushaertl) [inline .reuse/dep5:2]:
```suggestion
Upstream-Name: groupfolders
```

File: .reuse/dep5 [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+Upstream-Contact: Carl Schwan <carl@carlschwan.eu>
+Source: https://github.com/nextcloud/profiler
```
Comment (juliushaertl) [inline .reuse/dep5:4]:
```suggestion
Source: https://github.com/nextcloud/groupfolders
```

File: README.md (hunk 1 above)
Comment (juliushaertl) [reply README.md:8]: Would also be curious what the way would be to update this automatically from the git history. :)

**Comment by skjnldsv** [comment]
Addressed in subsidiary PR already :)

**Pull request closed without merging**